examples/api/test/material/menu_anchor/menu_anchor.3_test.dart

```diff
@@ -0,0 +1,108 @@
+
+      // Tap the menu button to open the menu.
+      await tester.tap(menuButton);
+      await tester.pumpAndSettle();
+
+      // Verify that the menu is open.
+      expect(find.text('Revert'), findsOneWidget);
+
+      // Tap the menu button again to close the menu.
+      await tester.tap(menuButton);
+      await tester.pumpAndSettle();
+
+      // Verify that the menu is closed.
+      expect(find.text('Revert'), findsNothing);
+    });
+
+    testWidgets('Menu items are tappable', (WidgetTester tester) async {
+      final Completer<void> revertCompleter = Completer<void>();
+
+      // Build the SimpleCascadingMenu widget with the callback.
+      await tester.pumpWidget(
+        MaterialApp(
+          home: Scaffold(
+            appBar: AppBar(
+              actions: <Widget>[
+                MenuAnchor(
+                  childFocusNode: FocusNode(debugLabel: 'Menu Button'),
+                  menuChildren: <Widget>[
+                    MenuItemButton(
+                      key: const ValueKey<String>('revertButton'),
+                      child: const Text('Revert'),
+                      onPressed: () {
+                        revertCompleter.complete();
+                      },
+                    ),
+                    MenuItemButton(
+                      child: const Text('Setting'),
+                      onPressed: () {},
+                    ),
+                    MenuItemButton(
+                      child: const Text('Send Feedback'),
+                      onPressed: () {},
+                    ),
+                  ],
+                  builder: (_, MenuController controller, Widget? child) {
+                    return IconButton(
+                      key: const ValueKey<String>('menuButton'),
+                      onPressed: () {
+                        if (controller.isOpen) {
+                          controller.close();
+                        } else {
+                          controller.open();
+                        }
+                      },
+                      icon: const Icon(Icons.more_vert),
+                    );
+                  },
+                ),
+              ],
+            ),
+          ),
+        ),
+      );
+
+      // Find the menu button.
+      final Finder menuButton = find.byKey(const ValueKey<String>('menuButton'));
+      expect(menuButton, findsOneWidget);
+
+      // Tap the menu button to open the menu.
+      await tester.tap(menuButton);
+      await tester.pumpAndSettle();
+
+      // Find the "Revert" menu item.
+      final Finder revertItem = find.byKey(const ValueKey<String>('revertButton'));
+      expect(revertItem, findsOneWidget);
+
+      // Tap the "Revert" menu item.
+      await tester.tap(revertItem);
+      await tester.pumpAndSettle();
+
+       // Verify that the "Revert" callback was called and completed.
+      await revertCompleter.future;
+      expect(revertCompleter.isCompleted, isTrue);
+    });
```

Comment:
Last step before approving this PR: consider removing this test because it does not call the example app. It is testing that a `MenuItemButton` is tappable, one can assume that this is already tested on the framework side (If the example app was defining particular callbacks it would make sense to check that they are called, but the example just use empty callbacks).